.gitignore

```diff
@@ -21,3 +21,9 @@
 /yarn-error.log
 
 .byebug_history
+
+# Ignore application configuration
+/config/application.yml
+
+# Ignore environment variables
```

Comment:
You don't need two of these rows, but it won't cause a problem (you can fix later)